flink-runtime/src/main/java/org/apache/flink/runtime/executiongraph/ExecutionGraph.java

```diff
@@ -306,6 +306,12 @@ public ExecutionGraph(
 		this.timeout = timeout;
 
 		this.restartStrategy = restartStrategy;
+
```

Comment:
@StephanEwen @uce was there a reason for not reading the executionConfig at this point already?